### PR TITLE
ci(chore): add script to manage drivers

### PR DESCRIPTION
**Pull request by brad-decker**
## **Description**
in #21481 we removed the devDeps for chromedriver and geckodriver, and used selenium manager to download binaries for browsers in CI. This removed our ability to run e2e tests locally. This PR adds a script to manage drivers and uses that in circleci.

Also adds a new command `yarn test:e2e:updatedrivers` which will install the correct versions of chromedriver and geckodriver  based on your installed browsers. 

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by danjm — DISMISSED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -1061,7 +1061,7 @@ jobs:
           command: mv ./builds-test ./builds
       - run:
           name: Activate Selenium Manager for Firefox
-          command: node_modules/selenium-webdriver/bin/linux/selenium-manager --browser firefox
+          command: yarn
```
Comment (jiexi) [inline .circleci/config.yml:1064]: is this right?

Comment (brad-decker) [reply .circleci/config.yml]: Nope, must have somehow escaped out of multicursor when typing

File: development/webdriver-manager.js
Comment (jiexi) [inline development/webdriver-manager.js:1]: needs to be renamed to `update-webdriver.js` or package.json needs to have paths updated

**Review by MALIKSHAHID786 — APPROVED** [review]

**Review by pedronfigueiredo — APPROVED** [review]

**Comment by danjm** [comment]
I saw an error when I first ran the script:
```
danjm@pop-os:~/metamask/metamask-extension$ yarn test:e2e:updatedrivers
WARN	The chromedriver version (119.0.6045.105) detected in PATH at /usr/bin/chromedriver might not be compatible with the detected chrome version (116.0.5845.140); currently, chromedriver 116.0.5845.96 is recommended for chrome 116.*, so it is advised to delete the driver in PATH and retry
INFO	Driver path: /usr/bin/chromedriver
INFO	Browser path: /usr/bin/google-chrome
INFO	Driver path: /home/danjm/.cache/selenium/geckodriver/linux64/0.33.0/geckodriver
INFO	Browser path: /usr/bin/firefox
```
And so still couldn't run e2e tests.

Then running `rm -rf /usr/bin/chromedriver` and re-running `yarn test:e2e:updatedrivers` solved the problem.

Also, not everyone will know about the scripts these PRs add. Maybe add a little bit of documentation with this PR?

**Comment by brad-decker** [comment]
i'm going to hold off on this PR as @HowardBraham has some ideas on how to make the driver update seamless for everyone.

**Comment by brad-decker** [comment]
No longer needed. Thanks @HowardBraham !

**Pull request closed without merging**